pump/storage/log.go

```diff
@@ -108,14 +108,12 @@ func (r *Record) isValid() bool {
 func newLogFile(fid uint32, name string) (lf *logFile, err error) {
 	fd, err := os.OpenFile(name, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
 	if err != nil {
-		err = errors.Trace(err)
-		return
+		return nil, errors.Annotatef(err, "open file %s failed", name)
```

Comment:
error of file operation contains complete information,  like `https://golang.org/src/os/file_unix.go?h=openFileNolog#L190`, don't need annotates here